airbyte-integrations/connectors/source-amplitude/source_amplitude/manifest.yaml

```diff
@@ -0,0 +1,171 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_pointer:
+        - "{{ options.get('data_field') }}"
```

Comment:
Using the latest version of the airbyte-cdk, `options` has been renamed `parameters`

Reply:
updated